packages/tooltip/src/Tooltip2D.js

```diff
@@ -37,6 +37,7 @@ export default function Tooltip2D(props) {
   } = props;
 
   const classes = useStyles();
+  const tooltipId = useId();
```

Comment:
See above comment about IDs without corresponding references in other properties